R/dbs.R

```diff
@@ -1,10 +1,9 @@
 dbs <- function(x, N, S, log = FALSE){
-  if (!all(is.finite(c(N, S)))) stop("all parameters should be finite")
-  if (N <= 0)  stop("N must be larger than zero")
-  if (S <= 0)  stop("S must be larger than zero")
-  if(any(x < 1)) stop("at least one x less than one")
```

Comment:
The current version makes also returns an error when x is outside the sample space. My fault, sorry.